### PR TITLE
Update asciinema theme according to new design #79

### PR DESCRIPTION
**Pull request by Pallavi-PH**
Signed-off-by: Pallavi-PH <pallaviph02@gmail.com>

In this PR,

1. We are updating asciinema code theme according to the new design
2. We are handling responsiveness of the ascinema code block across different devices

**Desktop and laptop:**

Screenshot 2021-05-17 at 5 11 46 PM

Screenshot 2021-05-17 at 5 12 17 PM

Screenshot 2021-05-17 at 5 12 36 PM

**Tablet:**

Screenshot 2021-05-17 at 5 44 22 PM

**Mobile:**

Screenshot 2021-05-17 at 5 43 45 PM

### CODE REVIEW

File: public/css/custom-asciinema-player.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .asciinema-terminal {
-  height: 320px !important;
+  height: 340px !important;
```
Comment (satyapriyamishra222) [inline public/css/custom-asciinema-player.css:2]: Is there any thing we can do about the !important declarations being used in this entire file ?

Comment (Pallavi-PH) [reply public/css/custom-asciinema-player.css:2]: Asciinema uses inline CSS. In order to override that we need to use `!important`

Comment (satyapriyamishra222) [reply public/css/custom-asciinema-player.css:2]: okay.

**Review by satyapriyamishra222 — APPROVED** [review]

**Review by IsAmrish — APPROVED** [review]
Lgtm

**Pull request merged**